api/README.md

```diff
@@ -218,16 +218,19 @@ Returns the information about the token referenced by the provided ID and chain
 | 500 | `error getting last block number from web3 endpoint` | 5021 | 
 
 ### GET `/tokens/{tokenID}/holders/{holderID}?chainID={chainID}&externalID={externalID}`
-Returns if the holder ID is already registered in the database as a holder of the token ID and chain ID provided, the external ID is optional.
+Returns the holder balance if the holder ID is already registered in the database as a holder of the token ID and chain ID provided, the external ID is optional.
 
 > `chainID` URL parameter is *mandatory*.
 
 > `externalID` URL parameter is *optional* by default, but required for external provided tokens like POAPs.
 
 - 📥 response:
 
-```
-true|false
+```json
+{
+    "holderID": "0x1234",
```

Comment:
@lucasmenendez  why do you retairned the holderId provided?

Reply:
for no reason, just to give more information

Reply:
do you suggest to return just the balance? ex:

```json
{
    "balance": "1234567890"
}
```